packages/components/package.json

```diff
@@ -30,10 +30,12 @@
 	],
 	"dependencies": {
 		"@babel/runtime": "^7.13.10",
-		"@emotion/cache": "^10.0.27",
-		"@emotion/core": "^10.1.1",
-		"@emotion/css": "^10.0.22",
-		"@emotion/styled": "^10.0.23",
+		"@emotion/cache": "^11.1.3",
+		"@emotion/core": "^11.0.0",
```

Comment:
Why do we still declare `@emotion/core` as a dependency? According to the migration docs, we should replace every usage with `@emotion/react` — and we are also updating its alias for storybook. There shouldn't be any need for installing it as a dependency?

Reply:
Hmmm, I think I was just pinning all the dependencies. I'll remove it and see if things still work correctly 🙂

Reply:
Done  in 7e56a9b